scss/modules/_filters.scss

```diff
@@ -20,25 +20,34 @@
 //
 // Styleguide modules.filters
 
+// by default show filters offscreen
 .filters {
   @include clearfix();
   @include u-bg--neutral();
+  @include transition(left 1s ease-in-out);
+  position: absolute;
   bottom: 0;
-  height: 0;
-  top: 0;
+  height: auto;
+  top: -20px;
+  left: -800px;
+  z-index: 10;
```

Comment:
We use variables for z-indexes. They're all in `_variables.scss` at the bottom. You'll want to choose one that's greater than the one used for $z-overlay` because that's the loading state overlay, which is currently showing up on top of the filter panel when it's open.